.github/renovate.json

```diff
@@ -1,20 +1,27 @@
 {
-  "schedule": ["every weekend"],
-  "labels": ["dependencies"],
+  "extends": [ "config:base", ":dependencyDashboardApproval" ],
+  "constraints": {
+    "node": "< 16"
+  },
+  "labels": [ "dependencies" ],
   "separateMajorMinor": "false",
+  "ignorePaths": [
+    "native-module/",
+    "spec/fixtures/",
+    "templates/"
+  ],
   "packageRules": [
     {
-      "matchDepTypes": ["devDependencies"],
-      "matchUpdateTypes": ["major", "minor", "patch", "pin", "digest", "lockFileMaintenance", "rollback", "bump"],
-      "groupName": "devDependencies",
-      "semanticCommitType": "chore",
-      "automerge": true
+      "matchDepTypes": [ "devDependencies" ],
+      "description": "Dev Dependencies",
+      "groupName": "DevDependencies",
+      "matchPaths": [ "package.json" ]
     },
     {
-      "matchDepTypes": ["dependencies"],
-      "matchUpdateTypes": ["major", "minor", "patch", "pin", "digest", "lockFileMaintenance", "rollback", "bump"],
-      "groupName": "dependencies",
-      "semanticCommitType": "fix"
+      "matchDepTypes": [ "dependencies" ],
+      "description": "Dependencies",
+      "groupName": "Dependencies",
+      "matchPaths": [ "package.json" ]
     }
   ]
```

Comment:
This stuff groups all `devDependencies` bumps and all `dependencies` bumps into one PR each for all relevant deps, if I'm not mistaken?

I dunno how many items we're expecting to get in the dashboard, if it's a lot this trade-off sort of makes sense, but I would expect some of these we can bump easily, while others we can't. Dunno if it breaks Renovate if we heavily edit one of their PRs to bump only the things that don't break? Maybe worth separating them out (though this would create a _ton_ of noise in this repo if we actually triggered/merged each of these deps as individual PRs...)

Maybe the dashboard should be informational, and manually bump however many things don't break on a branch and do a manual PR that way?

Reply:
Hmm this is a great point, we can remove the `packageRules` portion and let each dep be reported as a standalone item. That way we could us it for bumps if we wanted too, but it can still be easily used informationally.